game_api.txt

```diff
@@ -155,7 +155,8 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
+	texture = "default_wood.png", -- `backface_culling` will automatically be
+	-- set to `true` if not specified.
```

Comment:
Sorry, my comments have been assuming this is code.
See earlier in this doc to be consistent, you can use spaces to align this comment with the comment above.